src/runtime_src/core/tools/common/ReportAsyncError.cpp

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright (C) 2020 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+// ------ I N C L U D E   F I L E S -------------------------------------------
+// Local - Include Files
+#include <ctime>
+#include "ReportAsyncError.h"
+#include "core/common/query_requests.h"
+#include "core/common/device.h"
+#include "core/common/api/error_int.h"
+#include "core/include/experimental/xrt_error.h"
+#include "core/include/xrt_error_code.h"
+#include <boost/algorithm/string.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/date_time/posix_time/posix_time_io.hpp>
+namespace qr = xrt_core::query;

```

Comment:
Being that there isn't many query calls and this file does quality the namespaces, would you be so kind as to remove this entry.   Doing so will help future developers quickly find and "know" what domain a method is associated with.

Reply:
fixed